code/notificationbuilder/src/main/java/com/adobe/marketing/mobile/notificationbuilder/internal/extensions/NotificationCompatBuilderExtensions.kt

```diff
@@ -91,7 +90,8 @@ private fun NotificationCompat.Builder.setSmallIconColor(
 
     try {
         // sets the icon color if provided
-        setColorized(true).color = Color.parseColor("#$iconColorHex")
+        val color = Color.parseColor("#$iconColorHex")
```

Comment:
nit: was the `val` added for readability? its not used elsewhere right?

Reply:
`setColorized(true)` was called even if parsing the color failed. So setting the parsed value to a color to trigger the exception for invalid values